### PR TITLE
GCP: Running Jupyter

### PR DESCRIPTION
**Pull request by nikolayilyin**

### CODE REVIEW

**Comment by nikolayilyin** [comment]
test!

File: gcp/build.gradle [hunk 1]
```diff
@@ -70,6 +72,51 @@ tasks.register('deployToGCE') {
...
+        "shutdown_wait": "${ext.getParameterValue('jupyterShutdownWait') ?: 60 * 8}",
+        "storage_size": "${ext.getParameterValue('storageSize')}",
+        "shutdown_behaviour": "${ext.getParameterValue('shutdownBehaviour')}",
+        "storage_publish": ${"false".equalsIgnoreCase(ext.getParameterValue('s3Backup')) ? "false" : "true"}
+        }"""
+        logger.warn(pload)
+        HttpResponse result = makeJsonPostRequest("https://us-central1-beam-core.cloudfunctions.net/deploy_beam", pload)
+        logger.warn("response status: ${result.statusCode}, response message: ${result.statusMessage}, payload: ${result.content}")
+    }
+}
```
Comment (dimaopen) [inline gcp/build.gradle:94]: Is that possible to just reuse `deployToGCE` task with correctly defined params (run_beam, run_jupyter)?

**Review by dimaopen — COMMENTED** [review]
Just a single comment to consider.

**Comment by nikolayilyin** [comment]
test!

**Review by dimaopen — APPROVED** [review]

**Pull request merged**